### ISSUE TITLE
[All] Removal of interactive music playback

### ISSUE THREAD

**Issue by xan1242** [xan1242/XNFSMusicPlayer#15]
I'm considering removing the interactive music playback feature and I'll need to gather some thoughts and opinions on this matter...

The reasons why I'm considering it:

- The advancements in modding in-game files had grown since this plugin was made (MPFmaster)
- In its current form, it makes no sense. As soon as you modify the map file (mpf) parts of the original interactive music, this player automatically becomes invalid. This is because it relies on just playing back the samples instead of actually playing the music
- It's a big, complicated piece of code that just complicates matters, so limiting the scope to the EA Trax Jukebox will make things MUCH easier
- Removes the need for the "installer" which just extracts the samples

While it was cool to see the in-game dialogs control external utilities and make BASS play samples, I don't think it's worth keeping.

There is a possibility where I keep it, and that is only and only if someone manages to:

- Make an entirely custom Pathfinder player
- Or maybe, make this into a dumb sample player (but this would mean re-extracting the samples every time you modify something, kinda like what it is now)

As a feature, keeping it as-is I could also do (and just disabling it) but during the process of simplification, I might break it.

**Comment by alexpado** [comment]
Hi,

So this means that we won't have pursuit music anymore ? This would be a massive lost to me tbh. 

I love the fact I can just put my custom music for the races & open world and when entering a pursuit, the cool chase music starts.

**Comment by xan1242** [comment]
No, this simply means that the standard player (SndStream) would play it normally instead of BASS. You wouldn't lose the actual functionality. In fact, I'd argue it's more flexible this way.

**Comment by alexpado** [comment]
Well, that's okay with me, having a lot of files extracted from the game for "nothing" isn't ideal, I have to agree.
